src/store/model-store.js

```diff
@@ -49,6 +49,16 @@ class ModelStore extends Store {
     getModelByActionId(actionId) {
         return modelsByActionId[actionId];
     }
+
+    deleteModelByAddress(address){
```

Comment:
NACK: Store methods should only return data, never manipulate it.

If you'd like to use a "helper function" like this, move it outside of the store so that other modules can't modify the store data.